src/templates/basic.ts

```diff
@@ -2,7 +2,14 @@ import { Template } from './template';
 
 const basicTemplate: Template = {
   name: 'basic',
-  dependencies: ['husky', 'tsdx', 'tslib', 'typescript'],
+  dependencies: [
+    'husky',
+    'tsdx',
+    'tslib',
+    'typescript',
+    'size-limit',
+    '@size-limit/preset-small-lib',
```

Comment:
Does this not need any configuration to use? The docs didn't specify how to add presets to the config

Reply:
A default configuration is added in `package.json` file and IMO there's no need to add more stuff in this case. This could be edited by users if they want to

Reply:
The `package.json` config doesn't reference `@size-limit/preset-small-lib` though -- does it automatically detect it or something?

Reply:
Yep! you don't need to specify that you are using `@size-limit/preset-small-lib`, you just need to install it (check [docs](https://github.com/ai/size-limit#small-libraries))

Reply:
Thanks for the confirmation. Seems like some new docs were added that specify the step-by-step and don't mention configuring which confirms, but would be great if the docs explicitly stated that 😕 . Might file a PR for that